### PR TITLE
prop to hide nav from screen readers, combine a11y props

### PR DESCRIPTION
**Pull request by nelsonwong1012**
Prop to hide nav dots from screen reader software.

Combined accessibility related props into a single prop and set defaults.

### CODE REVIEW

File: src/propTypes.js [hunk 1]
```diff
@@ -62,11 +65,14 @@ export const propTypes = {
 }
 
 export const defaultProps = {
+  accessibilityOptions: {
+    closeButtonAriaLabel: 'Close',
```
Comment (nelsonwong1012) [inline src/propTypes.js]: @elrumordelaluz Need some help with this one.

```
accessibilityOptions: PropTypes.shape({
    ariaLabelledBy: PropTypes.string,
    closeButtonAriaLabel: PropTypes.string,
    showNavigationScreenReaders: PropTypes.bool,
  }),
export const defaultProps = {
  accessibilityOptions: {
    closeButtonAriaLabel: 'Close',
    showNavigationScreenReaders: true,
  },
<Component
    accessibilityOptions={{ ariaLabelledBy: 'some-element-id' }}
```
The code above would blow away the default values. Do you know of a clean way to be able to override individual keys in this object without blowing away other default keys?

Comment (elrumordelaluz) [reply src/propTypes.js]: What do you think of something like [this](https://codesandbox.io/s/condescending-kalam-xzdlx?fontsize=14&hidenavigation=1&theme=dark)

Comment (nelsonwong1012) [reply src/propTypes.js]: That totally works. Don't know why I didn't think of that. Thanks!

**Comment by elrumordelaluz** [comment]
Cool @nelsonwong1012, thanks for all your contributions!

**Pull request merged**